README.adoc

```diff
@@ -127,12 +140,15 @@ followed by fields that correspond to metadata fields.
 *The `description` field is optional and describes the purpose of the metric
 annotation.
 
-The `@Counted` annotation applies to the `list()` method. This annotation counts how many times you request the list of systems methods by visiting `http://localhost:9080/inventory/systems` to view the current contents of the
-inventory.
-* The `absolute` field makes the name of this annotation take the name of the method.
+The `@Counted` annotation applies to the `list()` method. This annotation counts how many times
+you request the list of systems methods by visiting `http://localhost:9080/inventory/systems` to
+view the current contents of the inventory.
+* The `absolute` field gives this annotation the name of the method.
 * The `description` field is optional and describes the purpose of the metric annotation.
-* The default value of the `monotonic` field is `false`. When `false`,
-the annotation counts the current invocations of the annotated method, and the counter is incremented before the annotated method returns. However, set the value to `true` instead.
+* The `monotonic` field is set to `true` so that the counter will only increment.
```

Comment:
How about this?

The `monotonic` field is set to `true` so that only the counter increments.

Reply:
Done

Reply:
It is saying that there is something else that can be incremented

Reply:
How about this?
The `monotonic` field is set to `true` so that the counter alone will increment.